src/api.py

```diff
@@ -196,19 +198,39 @@ def recitation(self):
                 else:
                     raise EditorNotFound()
             except EditorNotFound:
-                print("Text editor not found; set the 'EDITOR'" +
+                if platform.system() == "Windows":
+                    editor = "notepad"
+                    windowsfilename = f"{self.target.reference}"
+                    windowsfilename = windowsfilename.replace(":","-")
```

Comment:
[NIT] Instead of calling this across two lines, you can chain the replace command:

```suggestion
                    windowsfilename = f"{self.target.reference}".replace(":", "-")
```